.github/workflows/legacy-os.yml

```diff
@@ -16,3 +16,13 @@ jobs:
         run: |
           { { ! sudo scripts/lib/upgrade-zulip-stage-2 2>&1 >&3; } | tee upgrade.err; } 3>&1 >&2
           grep -Fq 'upgrade-zulip-stage-2: Unsupported platform: ubuntu 16.04' upgrade.err
+
+      - name: Report Status
```

Comment:
We always use [sentence case](https://zulip.readthedocs.io/en/latest/translating/translating.html#capitalization), Not Title Case.